content/en/docs/reference/config/proxy_extensions/stats/index.html

```diff
@@ -54,7 +54,7 @@ <h2 id="MetricConfig">MetricConfig</h2>
 </td>
 </tr>
 <tr id="MetricConfig-tags_to_remove">
-<td><code>tagsToRemove</code></td>
+<td><code>tags_to_remove</code></td>
```

Comment:
This change comes from a file in another repo. Making the change here will be overwritten the next day.

I think the update might need to be here: https://github.com/istio/proxy/blob/master/extensions/stats/config.proto#L44

Reply:
I believe that comes from here: https://github.com/istio/proxy/blob/master/extensions/stats/config.proto#L44

I don'r know enough about the generation from the .proto to the .pb.html.